process/block/metablock.go

```diff
@@ -588,7 +587,7 @@ func (mp *metaProcessor) RestoreBlockIntoPools(headerHandler data.HeaderHandler,
 	mp.blockTracker.RemoveLastNotarizedHeaders()
 
 	if metaBlock.IsStartOfEpochBlock() {
-		mp.epochStartTrigger.Revert(metaBlock.GetRound())
+		mp.epochStartTrigger.Revert(metaBlock)
```

Comment:
This line will do nothing, because the metaTrigger has a check for IsStartOfEpochBlock

Reply:
it is only for double protection.

Reply:
ok